src/main/java/imageData/Seams.java

```diff
@@ -0,0 +1,121 @@
+package imageData;
+
+public class Seams {
+    double[][] horizontalEnergyArray;
+    double[][] verticalEnergyArray;
+    int[] removeSeam;
+
+    public Seams(Energy energy) {
+        createVerticalEnergyArray(energy);
+        createHorizontalEnergyArray(energy);
```

Comment:
If this class calculates the vertical and horizontal energies, then what does the Energy class do?